lib/js/components/SelectList/SelectListItem/SelectListItem.vue

```diff
@@ -25,41 +31,55 @@
 
 	background-color: $color-neutral-background-ghost;
 	color: $color-neutral-text-heavy;
+	cursor: pointer;
 	display: flex;
+	justify-content: space-between;
```

Comment:
does it change anything? If yes, then it's probably unexpected

Reply:
Nah, I've added it before `flex-grow: 1` and didn't check if it's still needed.